plugin/phpactor.vim

```diff
@@ -376,7 +376,6 @@ function! phpactor#_rpc_dispatch(actionName, parameters)
     " >> open_file
     if a:actionName == "open_file"
         call phpactor#_switchToBufferOrEdit(a:parameters['path'])
-        exec ":edit"
```

Comment:
This was here for a reason probably - for cases where the file _has_ been modified on disk and should be reloaded.

Reply:
cases where this might happen:

- Replace references in current file
- Generate method in new file
- ??

Works on all of the above

Reply:
.. and if files _have_ been changed on disk, VIM should warn about this.